### PR TITLE
Add luigi 2.0.0 and python-daemon 2.0.6

### PR DESCRIPTION
**Pull request by kwilcox**
These are OSX/Linux only

### CODE REVIEW

**Comment by ocefpaf** [comment]
Kyle I will be offline for a while and I cannot merge this using my
not-so-smart the phone.

Feel free to merge it or ping Rich.

**Comment by kwilcox** [comment]
How do I go about skipping windows builds?  I left out `bld.bat` in hopes that would do the trick but AppVeyor is still trying to import.  Should I add the `nowin` comment to the import and test sections of `meta.yml`?

**Comment by ocefpaf** [comment]
Take a look at the nco meta.yaml. You need to add the "skip build" part.

File: luigi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    name: luigi
+    version: "2.0.0"
+
+source:
+    fn: luigi-2.0.0.tar.gz
+    url: https://pypi.python.org/packages/source/l/luigi/luigi-2.0.0.tar.gz
+    md5: 06258afcfcdd2f829167450fd5fed604
+
+build:
+    number: 0
```
Comment (rsignell-usgs) [inline luigi/meta.yaml]: add 
   skip: True  # [win]
here as in:
https://github.com/ioos/conda-recipes/blob/master/nco/meta.yaml#L9-L11

**Pull request merged**